### PR TITLE
MH-12883, Maven build of admin-ui module without frontend profile

### PR DESCRIPTION
**Pull request by sebastian-weiser**

### CODE REVIEW

File: modules/admin-ui/pom.xml [hunk 1]
```diff
@@ -417,8 +438,7 @@
           <instructions>
             <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
             <Build-Number>${buildNumber}</Build-Number>
-            <Private-Package>webapp.*</Private-Package>
-            <Include-Resource>{maven-resources}, target/grunt</Include-Resource>
+            <Include-Resource>{maven-resources}</Include-Resource>
```
Comment (lkiesow) [inline modules/admin-ui/pom.xml]: Not sure but isn't this the default so we could just omit this line completely?

**Review by lkiesow — APPROVED** [review]
Works great. Since this is ultimately made to speed up builds, can you also integrate this into the Travis configuration:

```diff
diff --git a/.travis.yml b/.travis.yml
index c6ca1b3c1..f4a1de374 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -87,11 +87,11 @@ jobs:
     - stage: build
       env: name=site
       before_script:
-        - mvn install -DskipTests=true -Dcheckstyle.skip=true -Pnone
+        - mvn install -DskipTests=true -Dcheckstyle.skip=true -P "none,!frontend"
       script:
         - >
            mvn site -Daggregate=true site:stage "-DstagingDirectory=$(pwd)/site" \
-             -Pnone \
+             -P "none,!frontend" \
              -DskipTests=true \
              -Dcheckstyle.skip=true \
              -Dpmd.skip=true \
```

Furthermore, can you maybe redirect this to `r/5.x`. It's probably less confusing for people if the tests work the same way on all active branches.

**Comment by sebastian-weiser** [comment]
removed unneccessary line in pom.xml and rebased it to 5.x

**Comment by lkiesow** [comment]
*Note that Sebastian is from Osnabrück (if someone was expecting me to merge this)*

**Pull request merged**